src/style.css

```diff
@@ -1,3 +1,106 @@
+html {
+  margin: 0;
+  font-family: 'Roboto ', sans-serif;
+  font-size: 12px;
+
+  --text-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.container {
+  display: flex;
+  position: relative;
+  align-items: center;
+  width: 1024px;
+  height: 60px;
+  justify-content: center;
+
+  text-transform: uppercase;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  position: absolute;
+  left: 50px;
+}
+
+.nav {
+  position: absolute;
+  right: 50px;
+  display: inline-block;
+  align-items: center;
+  justify-content: center;
+}
+
+.ul__nav {
+  list-style: none;
+  display: flex;
+  margin: 0;
+  padding: 0;
```

Comment:
Don't use tag names for styling (except 'html' and 'body'). Style all elements using '.class'. Instead of .ul__nav, use a class that describes the role or content of the list, such as .nav__list.